src/pages/Schedule.js

```diff
@@ -1,49 +1,170 @@
 import React, { Component } from 'react';
+import ScheduledMatch from '../components/ScheduledMatch';
 import provider from "../dataProvider";
 import Utils from "../utils";
 
 export default class Schedule extends Component {
   constructor(props) {
     super(props);
     this.state = {
-      dates: []
+      dates: [],
+      teams: [],
+      scheduledMatches: [],
+      filteredMatches: [],
+      dateOptionValue: '',
+      teamOptionValue: 'none'
     };
 
     this.fetchListOfPlayingDates();
+    this.fetchListOfScheduledMatches();
+    this.fetchListOfTeams();
+
   }
+  /** Data from Promise  */
 
   fetchListOfPlayingDates() {
     provider.getListOfPlayingDates().then(data => {
-      this.setState({ dates: data.matchDays });
+      const currentDate = new Date('July 2, 2018 23:15:30'); // Current date is chosen for testing purpose for now
```

Comment:
Please remove "test" parts of code before merging in master

Reply:
Ok ... can you please change the dates data ... because it contains year 2018

Reply:
no :) ... they will change once the new season is started